src/main/java/seedu/savenus/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,61 @@
+package seedu.savenus.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import javafx.collections.ObservableList;
+import javafx.collections.transformation.SortedList;
+import seedu.savenus.logic.commands.exceptions.CommandException;
+import seedu.savenus.model.Model;
+import seedu.savenus.model.food.Food;
+import seedu.savenus.model.food.FoodComparator;
+
+/**
+ * Sorts all the foods in the $aveNUS menu based on given criterion.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+    public static final String NO_ARGUMENTS_USAGE = "Note you have input in zero arguments:\n"
+            + "Example Usage: " + COMMAND_WORD + " PRICE ASC NAME DESC";
+    public static final String DUPLICATE_FIELD_USAGE = "Note you have input a duplicate field.";
```

Comment:
I think you should write "Note that you have entered a duplicate field."

Reply:
will do so in the next commit!